src/System.Windows.Extensions/tests/System/Media/SoundPlayerTests.cs

```diff
@@ -248,9 +248,22 @@ public void Play_NullStream_Success()
             player.Play();
         }
 
+        [Fact]
```

Comment:
Does this need to be conditioned for nano?

Reply:
yes